vllm/worker/model_runner.py

```diff
@@ -168,7 +167,91 @@ def from_broadcasted_tensor_dict(
 
 
 class ModelInputForGPUBuilder(ModelRunnerInputBuilderBase[ModelInputForGPU]):
-    """TBA"""
+    """Build ModelInputForGPU from SequenceGroupMetadata."""
+
+    # Compute function names for each sequence in
+    # a sequence group. The order of the functions matters
+    PER_SEQ_COMPUTE_FN_NAMES = (
```

Comment:
Can you add big warnings that the order should not be changed? (or each API should be written in order-agnostic way and add tests? )

Reply:
It's not likely to make them order-agnostic because some functions are updating/modifying fields based on existing values. I've added `WARNING` to the comment to make it more explicit.